src/dune_rules/gen_rules.ml

```diff
@@ -621,6 +628,10 @@ let gen_rules ctx sctx ~dir components : Gen_rules.result Memo.t =
       ~dir
       (Subdir_set.of_set (Filename.Set.of_list [ "cc_vendor" ]))
       (fun () -> Configurator_rules.gen_rules ctx)
+  | ".parameterized" :: rest ->
+    let* sctx = sctx
+    and* scope = Scope.DB.find_by_dir dir in
+    Parameterized_rules.gen_rules ~sctx ~scope ~dir rest
```

Comment:
This `~dir` is always the root of the workspace, in which case the scope is just the current project. Is that correct?